src/game/server/hl2/npc_zombine.cpp

```diff
@@ -30,6 +29,7 @@
 #include "asw_grenade_cluster.h"
 #include "asw_util_shared.h"
 #include "asw_marine.h"
+#include "npc_zombine.h"
```

Comment:
Traditionally, the include for the header file with the same name as the cpp file should come right after the one for cbase.h